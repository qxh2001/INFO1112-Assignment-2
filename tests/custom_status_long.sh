cd ..
python3 webserv.py config.cfg &
PID=$!
sleep 1
curl -I localhost:8070/cgibin/custom_status_long.py 2> /dev/null | grep HTTP | diff - tests/custom_status_long.out
kill $PID
