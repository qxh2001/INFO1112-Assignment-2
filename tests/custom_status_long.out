HTTP/1.1 303 This is a very long custom code
